src/styles/overrides/_google-search-results.scss

```diff
@@ -0,0 +1,138 @@
+/* Main Search Container */
+.gsc-control-cse {
+  padding: 0 !important;
+}
+
+/* Individual search result */
+.gsc-webResult.gsc-result {
+  border: $border-width-small solid $gray-light !important;
+  padding: $default-padding !important;
+  margin-bottom: $default-margin !important;
+
+  /* Search result title (it is a link) */
+  .gs-title {
+    @extend h3;
+
+    text-transform: uppercase;
+    font-size: $h2-font-size !important;
+    font-weight: 900;
```

Comment:
@theilman7 also changed this to $font-boldest

Reply:
That works.